code/game/gamemodes/wizard/spellbook.dm

```diff
@@ -568,7 +568,8 @@
 	else
 		user << "It appears to have no author."
 
-/obj/item/weapon/spellbook/proc/Initialize()
+/obj/item/weapon/spellbook/Initialize()

```

Comment:
Initialize is called on all atoms when SSobj is initialized. After that, it's called from atom/New. Also the spell book was